astroid/nodes/node_ng.py

```diff
@@ -275,7 +276,7 @@ def statement(self, *, future: Literal[True]) -> "nodes.Statement":
 
     def statement(
         self, *, future: Literal[None, True] = None
-    ) -> Union["nodes.Statement", "nodes.Module", NoReturn]:
+    ) -> Union["nodes.Statement", "nodes.Module", "NoReturn"]:
```

Comment:
If this PR should wait for the primer tests, maybe it's worth addressing the `NoReturn` issue separately before then.

Reply:
See #1252

Reply:
Left a small comment there. I think the separation does make sense weather we continue or wait with this one. Thanks for doing it!